README.md

```diff
@@ -114,8 +114,6 @@ MACHINE="jetson-tx2i"
 NVIDIA_DEVNET_MIRROR = "file:///home/$USER$/Downloads/nvidia/sdkm_downloads"
 
 CUDA_VERSION = "10.0"
```

Comment:
@Atomicchipmonk do you think it makes sense to also set `CUDA_VERSION` in the machine configs?

Reply:
yes indeed